NSS-Wacky-Eels/WorkforceManagement
# Tg list of employees

## Link to Ticket
Closes # [Link](https://github.com/NSS-Wacky-Eels/WorkforceManagement/issues/3)

## Description of Proposed Changes
- Added an employee insert to the Bangazon.sql
- Removed IsSupporvisor from the index file for listing employees
- Created an integration test for testing that a specific employee is in the list

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <tg-listEmp>
```
Run the new Bangazon.sql from this branch
Execute BangazonWorkforce and click on Employees to check that a list of Employees with only their name and department (Ex. Fred Jackson Music)
Then run the Get_IndexDisplayEmployees() integration test

## Impacted Areas in Application

* Employee/Index.cshtml
* Bangazon.sql
* IntegrationTests/EmployeeTests.cs

## Mentions @username

@KaylaReid
@alexfont321
@thatmikeparrish
@taylorlgulley


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
